docs/getting-started/what-is-dotnet.md

```diff
@@ -1,6 +1,150 @@
-# 🔧 What is the .NET Platform?
+# 🔧 What is the .NET Core Platform?
 
-> **Note**
-> 
-> This topic hasn’t been written yet! You can track the status of this issue through our public GitHub issue tracker. Learn how you can contribute on GitHub.
+.NET Core is a modular, cross-platform, open source implementation of the .NET Framework that runs on Windows devices, as well as on Linux and Mac OSX. Unlike the .NET Framework, .NET Core consists of a set of runtime and library components that can be used in various configurations, depending on the needs of your application.
+
+# .NET Core and the .NET Framework #
+To get a better sense of what .NET Core is, let’s compare it to the.NET Framework 4.x:
+<table>
```

Comment:
When I build this document locally using docfx, this table is very hard to read:

![Screenshot of the table](http://i.imgur.com/fPMiT8w.png)

I have already reported a similar issue at #418, but until that's fixed, maybe some local style could help?

The narrow width available for the table also doesn't help.